util/brew-update

```diff
@@ -62,7 +69,7 @@ defn releases(version)::
   MAC_INT:: "ys-${version}-macos-x64.tar.xz"
 
 defn check-version(version):
-  when version !~ /^0\.1\.\d+$/:
+  when version !~ /^\d(\.\d+){2}$/:
```

Comment:
Let's not generalize this quite yet. I purposefully coded it that way, because I wanted to remember to circle back here when I got to a major release.

Reply:
Maybe I need to add more comments in general :)

Reply:
Ok, `revert`ed…

> [!NOTE]
> For future reference, the general pattern here is missing a `+`, it should actually be `/^\d+(\.\d+){2}$/`.